src/style.scss

```
@@ -0,0 +1,126 @@
+$main-color: #060b35 ;
+$secondary-color: #616070 ;
+
+* {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", "Manrope", sans-serif;
+  color: $main-color;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  background-color: #fff;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px ;
+    height: 134px ;
+    margin: 0 auto;
+    margin-bottom: 40px;
+
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    width: 166px ;
+    height: 40px ;
+    display: flex;
+    justify-content: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background-color: #00acdc;
+    color: white;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+  }
+
+  &__buy:hover {
+    background-color: #f3f3f3;
+    color: #00acdc;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__review-box {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
```

```suggestion
  &__review-box {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 24px;
```
center elements vertically inside container